docs/develop/ruby/versioning.mdx

```diff
@@ -0,0 +1,164 @@
+---
```

Comment:
I'll be overhauling this page pretty significantly once this PR gets merged, FYI -- I need to propagate my Worker Versioning rewrites of the other SDKs to Ruby/PHP sometime hopefully next week.